src/Shared/PackageManagers/BaseProjectManager.cs

```diff
@@ -420,7 +420,6 @@ public virtual async Task<IPackageExistence> DetailedPackageVersionExistsAsync(P
         /// </summary>
         /// <param name="purl">The <see cref="PackageURL"/> to get the metadata for.</param>
         /// <param name="useCache">If the metadata should be retrieved from the cache, if it is available.</param>
-        /// <remarks>If no version specified, defaults to latest version.</remarks>
```

Comment:
Why remove this? Pretty sure this is still the case.

Reply:
The default implementation in BaseProjectManager.DetailedPackageVersionExistsAsync() is a call to PackageVersionExistsAsync(), which will exist if there is no version provided.